fgrpc/pingsrv.go

```diff
@@ -110,16 +111,21 @@ func PingServerTCP(port, cert, key, healthServiceName string, maxConcurrentStrea
 
 // PingClientCall calls the ping service (presumably running as PingServer on
 // the destination). returns the average round trip in seconds.
-func PingClientCall(serverAddr string, n int, payload string, delay time.Duration, tlsOpts *fhttp.TLSOptions) (float64, error) {
+func PingClientCall(serverAddr string, n int, payload string, delay time.Duration, tlsOpts *fhttp.TLSOptions, md metadata.MD,
```

Comment:
not your fault but this argument list is quite long

I already at some point move some common stuff in TLSOptions and wondering if we should have something else common for payload, n (called Exactly in http options), destination/serverAddr (though that last one could be in TLSOptions) - maybe CommonOptions or TLSAndWebOptions or something

(I realize this comment isn't exactly actionable, it's as much for me so I don't forget)

Reply:
Yes, I see that, why not just pass `GRPCRunnerOptions`?
```go
	options := fgrpc.GRPCRunnerOptions{
		TLSOptions: httpOpts,
		Destination: host.
		Payload: httpOpts.PayloadUTF8(),
		Delay.      *pingDelayFlag.
		Metadata: grpcMD().
	}
	_, err := fgrpc.PingClientCall(count, options)
```

Reply:
yeah not sure why I didn't do that in the first place - I suppose it might be because GRPCRunnerOptions includes RunnerOptions which is more for a full load test than a single call but it'd still be cleaner to ignore fields than pass so many

Reply:
will address (later/maybe) in #681